src/components/landing/Landing.module.scss

```
@@ -1,7 +1,7 @@
 @import "/styles/variables.scss";
 
 .landingContainer {
-  background-image: url("../../../public/1c0f146c96b2dc1b9ae15ff133378633.jpeg");
+  background-image: url("https://fd-internal-development.imgix.net/xn2YPAK6UgWU1-7YZ6fgvQkGmok.jpg");
```

Add a url param to make sure IMGIX is not using an origin image as those bandwidths cost a lot of money.
```suggestion
  background-image: url("https://fd-internal-development.imgix.net/xn2YPAK6UgWU1-7YZ6fgvQkGmok.jpg?q=45");
```